skabase/SKALogger/SKALogger.py

```diff
@@ -190,12 +190,16 @@ def SetCentralLoggingLevel(self, argin):
         central_logging_device = argin[1][:]
         i = 0
         while i < len(central_logging_level[:]):
-            self.info_stream("Central Logging level : %s, Device : %s",
-                             central_logging_level[i],
-                             central_logging_level[i])
-            dev_proxy = DeviceProxy(central_logging_device[i])
-            dev_proxy.centralLoggingLevel = central_logging_level[i]
-            i += 1
+            try:
+                self.info_stream("Central Logging level : %s, Device : %s",
+                                 central_logging_level[i],
+                                 central_logging_level[i])
+                dev_proxy = DeviceProxy(central_logging_device[i])
+                dev_proxy.centralLoggingLevel = central_logging_level[i]
+                i += 1
+            except DevFailed:
+                self.error_stream("Failed to set Central Logging level for [%s]", central_logging_device[i])
```

Comment:
Wouldn't it be nice to include the original exception's error message in the log?

Reply:
Added original exception message. Commit id 35e9e429f5c8d3afd470cdaa81ca011bedcdec83 addresses this.